src/cppcache/src/TcrMessage.hpp

```diff
@@ -1113,12 +1117,11 @@ class TcrMessageHelper {
       return NULL_OBJECT;
     } else if (!isObj) {
       // otherwise we're currently always expecting an object
-      char exMsg[256];
-      std::snprintf(exMsg, 255,
-                    "TcrMessageHelper::readChunkPartHeader: "
-                    "%s: part is not object",
-                    methodName);
-      LOGDEBUG("%s ", exMsg);
+
+      std::stringstream s;
+      s << "TcrMessageHelper::readChunkPartHeader: " << methodName << ": part is not object\n";
+      LOGDEBUG("%s ", s.str().c_str());
```

Comment:
Good question. I am still fascinated as to why std::snprintf isn't working correctly on Windows too.